.github/workflows/build.yml

```diff
@@ -20,35 +20,23 @@ jobs:
     timeout-minutes: 30
```

Comment:
I notice we have a env variable called `YARN_CACHE_FOLDER` on line 12 of this file, that points to `.cache/yarn`. This line should be deleted, as I don't see us _using_ the variable in the CI script anymore.

Reply:
Yes, please remove those yarn env values